src/Libraries/DSOffice/Excel.cs

```diff
@@ -82,10 +84,6 @@ public static Application RegisterAndGetApp()
             if (excel == null)
             {
                 excel = new Microsoft.Office.Interop.Excel.Application();
-                if (excel == null)
-                {
-                    throw new Exception("Excel could not be opened.");
-                }
```

Comment:
Oh sharks, you're right! Was having the impression that was a function call of some sort, duh! :sweat: